romanesco/plugins/gaia/requirements.txt

```diff
@@ -0,0 +1,11 @@
+numpy==1.9.2
+Shapely==1.5.11
+descartes==1.0.1
+cligj==0.3.0
+click==5.1
+click-plugins==1.0.1
+Fiona==1.6.1
+python-dateutil==2.4.2
+pandas==0.16.2
+pyproj==1.9.4
+geopandas==0.1.1
```

Comment:
At some point we may want to cache the python path on travis because these packages take a while to install.